frb_codegen/src/ir/ty_delegate.rs

```diff
@@ -48,6 +52,8 @@ impl IrTypeTrait for IrTypeDelegate {
                 "ZeroCopyBuffer_".to_owned() + &self.get_delegate().dart_api_type()
             }
             IrTypeDelegate::PrimitiveEnum { ir, .. } => ir.safe_ident(),
+            IrTypeDelegate::Backtrace => "String".to_owned(),
+            IrTypeDelegate::Anyhow => "Anyhow".to_owned(),
```

Comment:
In addition, shall we let `FrbAnyhowException implements FrbException`, since we already have a quite good exception parent-child class hierarchy now in this PR :)